events/interactionCreate.js

```diff
@@ -1,8 +1,28 @@
+const {commandRoles} = require('../config/slashCommandPermissions');
+
+function isAuthorized(interaction, commandRoles) {
+  const allowedRoles = commandRoles.get(interaction.commandName);
+  if (
+    allowedRoles[0] === '@everyone' ||
+    interaction.member.roles.cache.some((r) => allowedRoles.includes(r.id))
+  ) {
+    return true;
+  }
+  return false;
```

Comment:
Could just return the statement and not use a conditional here.

Reply:
Good catch. I made this small refactor.